source/common/http/header_map_impl.h

```diff
@@ -319,6 +319,8 @@ class HeaderMapImpl : NonCopyable {
   void insertByKey(HeaderString&& key, HeaderString&& value);
   static uint64_t appendToHeader(HeaderString& header, absl::string_view data,
                                  absl::string_view delimiter = ",");
+  static absl::string_view delimiterByHeader(const LowerCaseString& key,
```

Comment:
If this is not used outside of the cc file it can be a helper function in empty namespace

Reply:
get it.